Formula/yomo.rb

```diff
@@ -0,0 +1,19 @@
+class Yomo < Formula
+  desc "Streaming Serverless Framework for Low-latency edge computing applications"
+  homepage "https://yomo.run"
+  url "https://github.com/yomorun/yomo/archive/v0.4.1.tar.gz"
+  sha256 "309095eadd8f1d8230a79ad47aa20728edcde55089077f210f06fa8a11627760"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build"
+    bin.install "bin/yomo"
+  end
+
+  test do
+    assert_equal "Run a YoMo Serverless Function in development mode",
+    shell_output("#{bin}/yomo -h").lines.first.chomp
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
@SMillerDev Hi, We've update the `brew test` issue, please review, thanks.